datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/manifest.json

```diff
@@ -2,23 +2,40 @@
   "manifest_version": "2.0.0",
   "app_uuid": "{app_uuid}",
   "app_id": "{integration_id}",
-  "display_on_public_website": false,
+  "display_on_public_website": true,
```

Comment:
Just curious why we want to make this `true` by default now?

Reply:
We have partners change this to true immediately when they open the PR. Changing this aligns with what we are requesting from Partners

Reply:
@emarsha94 we have the opposite requirement for `integrations-core`, where we turn it to `true` weeks after the initial PR is merged to master.

We can parametrize this just like we already do for the changelog.

Reply:
@iliakur Could you clarify what you mean so I can better understand how to make the changes so that it works for both teams?